go.mod

```diff
@@ -59,14 +61,12 @@ require (
 	github.com/Masterminds/sprig/v3 v3.2.2 // indirect
 	github.com/PuerkitoBio/purell v1.1.1 // indirect
 	github.com/PuerkitoBio/urlesc v0.0.0-20170810143723-de5bf2ad4578 // indirect
-	github.com/antlr/antlr4/runtime/Go/antlr v0.0.0-20210826220005-b48c857c3a0e // indirect
```

Comment:
Surprised to see antlr in here, I wonder which component needs this

Reply:
I did some digging and found that it comes from https://github.com/google/cel-go